### ISSUE TITLE
Game crash upon starting campaign mission (sub just got attacked by ballast flora)

### ISSUE THREAD

**Issue by meatkixass** [Regalis11/Barotrauma#4617]
![Screen Shot 2021-01-01 at 9 08 08 PM](https://user-images.githubusercontent.com/76856986/103449603-650bf100-4c78-11eb-8bdc-cc84e86dff08.png)
- [X] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Upon starting the campaign mission in either single or multiplayer, the game crashes right after the mission loads up, but before any player can start moving. Right before starting up this mission, the submarine was docked into the station, and ballast flora was infecting the submarine. The flora was cleared up, and the pumps were repaired with no issues. It is only upon loading up the mission that the game crashes. To be specific, the current mission is Terminating the Hammerhead Swarm. This crash does not occur when I load up a different campaign save file/mission.

**Steps To Reproduce**
Allow ballast flora to infect pumps, clear flora, and start a new mission. The mission type may not be relevant, but selecting the Terminating the Hammerhead Swarm mission may reproduce the crash.

**Version**
v0.11.0.10
MacBook Air macOS Catalina ver. 10.15.3

**Additional [information**]
Crash report log:

Barotrauma Client crash report (generated on 1/1/2021 8:41:48 PM)

Barotrauma seems to have crashed. Sorry for the inconvenience! 

8936387FE9FEE612E8FF0923EE1A0C1B

Game version 0.11.0.10 (ReleaseMac, branch release, revision dc7157029e)
Graphics mode: 1440x900 (BorderlessWindowed)
VSync ON
Language: English
Selected content packages: Vanilla 0.9
Level seed: NoctaeMoulin
Loaded submarine: Typhon (529803956EB768957EBB44ABC168DD53)
Selected screen: Barotrauma.GameScreen
SteamManager initialized
Client (Round had started)

System info:
    Operating system: Unix 19.3.0.0 64 bit
    GPU name: 
    Display mode: {Width:1440 Height:900 Format:Color AspectRatio:1.6}
    GPU status: Normal

Exception: Failed to generate stream buffers: SoundChannel (/Users/calvinlee/Library/Application Support/Steam/steamapps/common/Barotrauma/Barotrauma.app/Contents/MacOS/Content/Sounds/Music/Layer 0 - Basic.ogg), Invalid value (System.Exception)
Target site: Void .ctor(Barotrauma.Sounds.Sound, Single, System.Nullable`1[Microsoft.Xna.Framework.Vector3], Single, Single, Single, System.String, Boolean)
Stack trace: 
   at Barotrauma.Sounds.SoundChannel..ctor(Sound sound, Single gain, Nullable`1 position, Single freqMult, Single near, Single far, String category, Boolean muffle) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundChannel.cs:line 543
   at Barotrauma.Sounds.Sound.Play(Nullable`1 gain, String category) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/Sound.cs:line 143
   at Barotrauma.SoundPlayer.UpdateMusic(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundPlayer.cs:line 879
   at Barotrauma.SoundPlayer.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundPlayer.cs:line 369
   at Barotrauma.GameMain.Update(GameTime gameTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/GameMain.cs:line 829
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime) in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 656
   at Microsoft.Xna.Framework.Game.Tick() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 504
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/SDL/SDLGamePlatform.cs:line 94
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 398
   at Microsoft.Xna.Framework.Game.Run() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 368
   at Barotrauma.Program.Main(String[] args) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Program.cs:line 59

Last debug messages:
[1/1/2021 8:41:46 PM] boof
[1/1/2021 8:41:46 PM] yaungmoney
[1/1/2021 8:41:46 PM] Lauzy
[1/1/2021 8:41:46 PM] hatrixo
[1/1/2021 8:41:46 PM] iDerplikeMen
[1/1/2021 8:41:46 PM] guywhoeats2much
[1/1/2021 8:41:45 PM] iDerplikeMen
[1/1/2021 8:41:45 PM] guywhoeats2much
[1/1/2021 8:41:37 PM] Task count: 0
[1/1/2021 8:41:33 PM] Generating an outpost for the end of the level... (Location: Moulin, level type: LocationConnection)
[1/1/2021 8:41:32 PM] Generating an outpost for the start of the level... (Location: Noctae, level type: LocationConnection)
[1/1/2021 8:40:25 PM] Lobby created!
[1/1/2021 8:40:24 PM] Lobby created!
[1/1/2021 8:40:24 PM] Lobby created!
[1/1/2021 8:40:24 PM] Lobby created!
[1/1/2021 8:40:24 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:23 PM] Lobby created!
[1/1/2021 8:40:22 PM] Lobby created!
[1/1/2021 8:40:22 PM] Lobby created!
[1/1/2021 8:40:22 PM] Lobby created!
[1/1/2021 8:40:22 PM] Lobby created!
[1/1/2021 8:40:22 PM] Lobby created!
[1/1/2021 8:40:21 PM] Lobby created!
[1/1/2021 8:40:21 PM] Lobby created!
[1/1/2021 8:40:21 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:20 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:19 PM] Lobby created!
[1/1/2021 8:40:18 PM] Lobby created!
[1/1/2021 8:40:18 PM] Lobby created!
[1/1/2021 8:40:18 PM] Lobby created!
[1/1/2021 8:40:18 PM] Lobby created!
[1/1/2021 8:40:18 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:17 PM] Lobby created!
[1/1/2021 8:40:16 PM] Lobby created!
[1/1/2021 8:40:16 PM] Lobby created!
[1/1/2021 8:40:16 PM] Lobby created!
[1/1/2021 8:40:16 PM] Lobby created!
[1/1/2021 8:40:16 PM] Lobby created!
[1/1/2021 8:40:15 PM] Lobby created!
[1/1/2021 8:40:15 PM] Lobby created!
[1/1/2021 8:40:15 PM] Lobby created!
[1/1/2021 8:40:15 PM] Lobby created!
[1/1/2021 8:40:15 PM] Lobby created!
[1/1/2021 8:40:14 PM] Lobby created!
[1/1/2021 8:40:14 PM] Lobby created!
[1/1/2021 8:40:14 PM] Lobby created!
[1/1/2021 8:40:14 PM] Lobby created!
[1/1/2021 8:40:14 PM] Lobby created!
[1/1/2021 8:40:13 PM] Lobby created!
[1/1/2021 8:40:13 PM] Lobby created!
[1/1/2021 8:40:13 PM] Lobby created!
[1/1/2021 8:40:12 PM] Lobby created!
[1/1/2021 8:40:12 PM] Lobby created!
[1/1/2021 8:40:12 PM] Lobby created!
[1/1/2021 8:40:12 PM] Lobby created!
[1/1/2021 8:40:11 PM] Lobby created!
[1/1/2021 8:40:11 PM] Lobby created!
[1/1/2021 8:40:11 PM] Lobby created!
[1/1/2021 8:40:11 PM] Lobby created!
[1/1/2021 8:40:11 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:10 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:09 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:08 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:07 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:06 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:05 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:04 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:03 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:02 PM] Lobby created!
[1/1/2021 8:40:01 PM] Lobby created!
[1/1/2021 8:40:01 PM] Lobby created!
[1/1/2021 8:40:01 PM] Lobby created!
[1/1/2021 8:40:01 PM] Lobby created!
[1/1/2021 8:40:01 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:40:00 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:59 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:58 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:57 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:56 PM] Lobby created!
[1/1/2021 8:39:55 PM] Lobby created!
[1/1/2021 8:39:55 PM] Lobby created!
[1/1/2021 8:39:55 PM] Lobby created!
[1/1/2021 8:39:55 PM] Lobby created!
[1/1/2021 8:39:55 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:54 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:53 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:52 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:51 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:50 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:49 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:48 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:47 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:46 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:45 PM] Lobby created!
[1/1/2021 8:39:44 PM] Lobby created!
[1/1/2021 8:39:44 PM] Lobby created!
[1/1/2021 8:39:44 PM] Lobby created!
[1/1/2021 8:39:44 PM] Lobby created!
[1/1/2021 8:39:44 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:43 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:42 PM] Lobby created!
[1/1/2021 8:39:41 PM] Lobby created!
[1/1/2021 8:39:41 PM] Lobby created!
[1/1/2021 8:39:41 PM] Lobby created!
[1/1/2021 8:39:41 PM] Lobby created!
[1/1/2021 8:39:41 PM] Lobby created!
[1/1/2021 8:39:39 PM] WARNING: GameSession set to null while in the game screen
   at System.Environment.get_StackTrace()
   at Barotrauma.GameMain.set_GameSession(GameSession value) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/GameMain.cs:line 82
   at Barotrauma.Networking.GameClient.Disconnect() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 2621
   at Barotrauma.Networking.GameClient.CancelConnect() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 441
   at Barotrauma.Networking.GameClient.OnDisconnect(Boolean disableReconnect) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 1020
   at Barotrauma.Networking.SteamP2POwnerPeer.Close(String msg, Boolean disableReconnect) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/Primitives/Peers/SteamP2POwnerPeer.cs:line 418
   at Barotrauma.Networking.GameClient.Disconnect() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 2587
   at Barotrauma.Networking.GameClient.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 614
   at Barotrauma.GameMain.Update(GameTime gameTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/GameMain.cs:line 951
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime) in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 651
   at Microsoft.Xna.Framework.Game.Tick() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 504
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/SDL/SDLGamePlatform.cs:line 94
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 398
   at Microsoft.Xna.Framework.Game.Run() in <DEV>/Libraries/MonoGame.Framework/Src/MonoGame.Framework/Game.cs:line 368
   at Barotrauma.Program.Main(String[] args) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Program.cs:line 59
[1/1/2021 8:39:39 PM] Error while reading a message from server. {Read error: please send us "crashreport_object.bin"!}
   at Barotrauma.Networking.GameClient.ReadIngameUpdate(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 2248
   at Barotrauma.Networking.GameClient.ReadDataMessage(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 746
   at Barotrauma.Networking.SteamP2POwnerPeer.HandleDataMessage(IReadMessage inc)
   at Barotrauma.Networking.SteamP2POwnerPeer.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/Primitives/Peers/SteamP2POwnerPeer.cs:line 227
   at Barotrauma.Networking.GameClient.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 614
[1/1/2021 8:39:39 PM] Writing object data to "crashreport_object.log", please send this file to us at http://github.com/Regalis11/Barotrauma/issues
[1/1/2021 8:39:39 PM]  - guywhoeats2much
[1/1/2021 8:39:39 PM]  - Metal Crate
[1/1/2021 8:39:39 PM]  - Coilgun Ammunition Box
[1/1/2021 8:39:39 PM]  - Crowbar
[1/1/2021 8:39:39 PM]  - Revolver
[1/1/2021 8:39:39 PM]  
[1/1/2021 8:39:39 PM]    at Barotrauma.Character.ClientRead(ServerNetObject type, IReadMessage msg, Single sendingTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/CharacterNetworking.cs:line 168
   at Barotrauma.Networking.GameClient.ReadIngameUpdate(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 2159
[1/1/2021 8:39:39 PM]  
[1/1/2021 8:39:39 PM] Previous object was 112 bits long (14 bytes)
[1/1/2021 8:39:39 PM] Previous header: ENTITY_POSITION
[1/1/2021 8:39:39 PM] Header: ENTITY_POSITION
[1/1/2021 8:39:39 PM] Read position: 6408
[1/1/2021 8:39:39 PM] Message length: 7112 (889 bytes)
[1/1/2021 8:39:39 PM] Object reference not set to an instance of an object.
[1/1/2021 8:39:39 PM] Received an inventory update message for an entity with no inventory (guywhoeats2much, removed: False)
[1/1/2021 8:39:39 PM] Failed to read event for entity "EntitySpawner"! {Failed to create OpenAL buffer for non-streamed sound: Invalid value}
   at Barotrauma.Sounds.Sound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/Sound.cs:line 172
   at Barotrauma.Sounds.OggSound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/OggSound.cs:line 61
   at Barotrauma.Sounds.SoundManager.LoadSound(String filename, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundManager.cs:line 349
   at Barotrauma.Submarine.LoadRoundSound(XElement element, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Map/Submarine.cs:line 133
   at Barotrauma.CharacterSound..ctor(SoundParams soundParams) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/CharacterSound.cs:line 24
   at Barotrauma.Character.<InitProjSpecific>b__350_0(SoundParams s) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/Character.cs:line 151
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at Barotrauma.Character.InitProjSpecific(XElement mainElement) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/Character.cs:line 153
   at Barotrauma.Character..ctor(String speciesName, Vector2 position, String seed, CharacterInfo characterInfo, UInt16 id, Boolean isRemotePlayer, RagdollParams ragdollParams) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Characters/Character.cs:line 879
   at Barotrauma.Character.Create(String speciesName, Vector2 position, String seed, CharacterInfo characterInfo, UInt16 id, Boolean isRemotePlayer, Boolean hasAi, Boolean createNetworkEvent, RagdollParams ragdoll) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Characters/Character.cs:line 796
   at Barotrauma.Character.ReadSpawnData(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/CharacterNetworking.cs:line 436
   at Barotrauma.EntitySpawner.ClientRead(ServerNetObject type, IReadMessage message, Single sendingTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/EntitySpawner.cs:line 34
   at Barotrauma.Networking.ClientEntityEventManager.Read(ServerNetObject type, IReadMessage msg, Single sendingTime, List`1 entities) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/NetEntityEvent/ClientEntityEventManager.cs:line 239
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4365)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Failed to spawn item "shotgunshell" in the inventory of "shotgun (4364)" (component index out of range). Index: 2, components: 2.
[1/1/2021 8:39:39 PM] Error while loading entity of the type Barotrauma.Items.Components.RangedWeapon. {Failed to create OpenAL buffer for non-streamed sound: Invalid value}
   at Barotrauma.Sounds.Sound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/Sound.cs:line 172
   at Barotrauma.Sounds.OggSound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/OggSound.cs:line 61
   at Barotrauma.Sounds.SoundManager.LoadSound(String filename, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundManager.cs:line 349
   at Barotrauma.Submarine.LoadRoundSound(XElement element, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Map/Submarine.cs:line 133
   at Barotrauma.Items.Components.ItemComponent.LoadElemProjSpecific(XElement subElement) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Items/Components/ItemComponent.cs:line 501
   at Barotrauma.Items.Components.ItemComponent..ctor(Item item, XElement element) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemComponent.cs:line 327
   at Barotrauma.Items.Components.RangedWeapon..ctor(Item item, XElement element) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Items/Components/Holdable/RangedWeapon.cs:line 68
[1/1/2021 8:39:39 PM] Error while loading entity of the type Barotrauma.Items.Components.RangedWeapon. {Failed to create OpenAL buffer for non-streamed sound: Invalid value}
   at Barotrauma.Sounds.Sound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/Sound.cs:line 172
   at Barotrauma.Sounds.OggSound.InitializeALBuffers() in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/OggSound.cs:line 61
   at Barotrauma.Sounds.SoundManager.LoadSound(String filename, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundManager.cs:line 349
   at Barotrauma.Submarine.LoadRoundSound(XElement element, Boolean stream) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Map/Submarine.cs:line 133
   at Barotrauma.Items.Components.ItemComponent.LoadElemProjSpecific(XElement subElement) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Items/Components/ItemComponent.cs:line 501
   at Barotrauma.Items.Components.ItemComponent..ctor(Item item, XElement element) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemComponent.cs:line 327
   at Barotrauma.Items.Components.RangedWeapon..ctor(Item item, XElement element) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Items/Components/Holdable/RangedWeapon.cs:line 68
[1/1/2021 8:39:39 PM] Failed to read event for entity "Kip Buckley"! {Object reference not set to an instance of an object.}
   at Barotrauma.Character.Kill(CauseOfDeathType causeOfDeath, Affliction causeOfDeathAffliction, Boolean isNetworkMessage, Boolean log) in <DEV>/Barotrauma/BarotraumaShared/SharedSource/Characters/Character.cs:line 3241
   at Barotrauma.Character.ReadStatus(IReadMessage msg) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/CharacterNetworking.cs:line 545
   at Barotrauma.Character.ClientRead(ServerNetObject type, IReadMessage msg, Single sendingTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Characters/CharacterNetworking.cs:line 344
   at Barotrauma.Networking.ClientEntityEventManager.Read(ServerNetObject type, IReadMessage msg, Single sendingTime, List`1 entities) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/NetEntityEvent/ClientEntityEventManager.cs:line 238
[1/1/2021 8:39:39 PM] Received an inventory update message for an entity with no inventory (Kip Buckley, removed: False)

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/4504
